src/types/venues.ts

```diff
@@ -50,6 +50,8 @@ export interface BaseVenue {
   name: string;
   slug: SpaceSlug;
   access?: VenueAccessMode;
+  subtitle?: string;
+  description?: string;
```

Comment:
are you sure we need these fields?
there's a description in `space?.config?.landingPageConfig?.description`. And about a subtitle. As it can't be added in the admin, I doubt it'll be used by anyone.

Reply:
removed in [09146a2](https://github.com/sparkletown/sparkle/pull/3074/commits/09146a2a8a1fd2ccbb351fd1056f0487e6181767)